### PR TITLE
[tools] Enable CI status pull requests

### PR DESCRIPTION
**Pull request by kangxu**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+language: python
+python:
+  - "2.7"
+script: ./tools/travis/lint.py
```
Comment (zqzhang) [inline .travis.yml]: Please put the script in `tools/format`

File: tools/travis/lint.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+# Authors:
+#         Xu, Kang <kangx.xu@intel.com>
+
+import sys
+import os
+import re
+import os.path
+import commands
+
+Lint_Root = os.path.dirname(os.path.abspath(__file__))
```
Comment (zqzhang) [inline tools/travis/lint.py]: Why not use https://github.com/w3c/wpt-tools/blob/master/lint/lint.py with some modification against testharness.js checking?

**Comment by kangxu** [comment]
@zqzhang 
I will update script refer to https://github.com/w3c/wpt-tools/blob/master/lint/lint.py

**Comment by zqzhang** [comment]
@kangxu thank you very much for trying this to fix #1680.

Because we cannot access to the setting of this repository, let's stop the task.

**Pull request closed without merging**